### PR TITLE
Added bengali cheatsheet link from the devsonket team.

### PR DESCRIPTION
**Pull request by shabab477**
Devsonket Team: https://github.com/orgs/devsonket/people

## What does this PR do?
Add Resource(s)

### Description 
The PR adds links to Bengali subsection for resources regarding cheat sheets in Bengali.

### Why is this valuable (or not)?
Helps new programmers with little English knowledge in Bangladesh find common commands with Bengali description.

### How do we know it's really free?
Its open source and I am in the open-source organisation team.

### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-programming-books-bl.md [hunk 1]
```diff
@@ -12,6 +13,11 @@
 * [Computer Programming](http://cpbook.subeen.com/p/blog-page.html) - Tamim Shahriar Subeen 
 
 
+### Developer Cheatsheet
+
+* [ডেভসংকেত: বাংলা চিটশিটের ভান্ডার](https://devsonket.com/) - Devsonket Team
```
Comment (SethiPandi) [inline free-programming-books-bl.md:18]: Please remove the trailing slash from root URLs!

```suggestion
* [ডেভসংকেত: বাংলা চিটশিটের ভান্ডার](https://devsonket.com) - Devsonket Team
```

**Review by SethiPandi — CHANGES_REQUESTED** [review]
You can get more guidance on the formatting and conventions of this repository by checking out the [contribution guide](https://www.github.com/EbookFoundation/free-programming-books/tree/master/CONTRIBUTING.md).

You can also click on the failed checks to get more information on any check failures. You'll be able to see exactly what was wrong and how to fix it!

File: free-programming-books-bl.md (hunk 1 above)
Comment (shabab477) [reply free-programming-books-bl.md]: Yes! Thank you, I have been looking so long for finding what's wrong!

**Comment by eshellman** [comment]
not usually a good idea to create a category for a single resource- unlikely that there will be additions to this category.  I suggest coming up with a new category name that would also include the "stack learners" link.

**Comment by shabab477** [comment]
>  I suggest coming up with a new category name that would also include the "stack learners" link.

I think something along the lines of **Professional Development** or **Software Engineering** or **Misc** sounds okay. What do you think?

**Pull request merged**